src/views/GovernanceTaskList/index.tsx

```diff
@@ -42,11 +43,81 @@ const GovernanceTaskList = () => {
   );
 
   const intakeStatus = intakeStatusFromIntake(systemIntake);
-  const intakeLink = chooseIntakeLink(systemIntake, intakeStatus);
+  const intakeLink = () => {
+    const link = chooseIntakeLink(systemIntake, intakeStatus);
+    switch (intakeStatus) {
+      case 'COMPLETED':
+        return <Link to={link}>View Submitted Request Form</Link>;
+      case 'CONTINUE':
+        return (
+          <UswdsLink className="usa-button" variant="unstyled" href={link}>
```

Comment:
The `href` prop here makes me think this is a plain ol' `a` tag, and not something that is using react router. Is that so? Using `Link` from React Router helps us maintain redux context between page views, rather than the browser treating the new URL as an entirely new instance of the application. There are distinct benefits to making sure we always use `Link` when the route is internal to our app.